### PR TITLE
Release notes 343

### PR DESCRIPTION
**Pull request by kaitlynmortimer**

### CODE REVIEW

File: content/release/v3/3.4/ReleaseNotes343.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+description: "Contrast 3.4.3 August 2017"
+tags: "3.4.3 August Release Notes"
+-->
+
+Set optimal standards for user accounts in the refreshed Security page, and create Protect policies before onboarding to any applications. Once you do, we'll give you some complimentary guidance on using your applications so you see the results you want.  
+
+## Fixes
+
+* No more SAML issues during authentication set up 
+* API documentation is ready to go from Contrast on Windows
```
Comment (SteveFeldman) [inline content/release/v3/3.4/ReleaseNotes343.md]: I would recommend removing this. It totally doesn't make any sense...

File: content/release/v3/3.4/ReleaseNotes343.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* XML errors are things of the past
+* Trace processing accepts trace events  
+* IP Blacklist page is back  
+* Protect events render again 
+
+## Improvements 
+ 
+* Use the revamped Security page to manage password policy, session timeouts, two-step verification and IP restrictions. You can also access an audit log, where you can search for past activity in your organization by keyword and date.
+
+* We no longer support instrumentation through the .NET Profiling API and now only use Duplex technology: a patented mechanism that allows us to coexist peacefully with your other favorite .NET agents.
```
Comment (SteveFeldman) [inline content/release/v3/3.4/ReleaseNotes343.md]: Can we move this to the top of Improvements?

**Comment by kaitlynmortimer** [comment]
@SteveFeldman Items have been removed/rearranged.

File: content/release/v3/3.4/ReleaseNotes343.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+* Change the severity for an entire vulnerability type with the click of a confirmation button. 
+
+## Agent Updates
+
+* **Java Agent Summary:** We added several rule accuracy improvements for Assess and Protect along with the ability to more easily deploy custom rules and better support for library detection in GlassFish. We also fixed a bug in the data flow engine that would prevent vulnerabilities from being reported in certain conditions.
+
+* **.NET Agent Summary:** We now report potential security controls and provide more reliable initialization. We also improved stability of IIS-Express instrumentation as well as Assess and Protect accuracy.
+
+* **Node.js Agent Summary:** We're supporting more useful logging for application clustering and automatically adding applications to existing groups. We fixed an issue that was preventing URLs from showing up in application activity - and, yes, it’s really fixed this time.
```
Comment (cixel) [inline content/release/v3/3.4/ReleaseNotes343.md]: Can "We're supporting" be "We've added" or "We added"? 
More useful logging for clusters isn't really something we support, it's just something we're doing more of/a better job at

**Pull request merged**